### DISCUSSION TITLE
Display positive/negative values on a stacked bar chart?

### DISCUSSION THREAD

**Discussion by djasnowski** [airbnb/visx#1624] [Q&A] [answered]
https://airbnb.io/visx/barstack

Is there a way to display negative values on a stack bar chart? I'm combing the documentation and not seeing anything besides use D3. I don't want to have to use D3 if I don't have too.

My example: https://codesandbox.io/s/interesting-rumple-vqou24

**Comment by williaster** [comment, marked as answer]
Hi! Yes see this issue/discussion https://github.com/airbnb/visx/issues/291

tl;dr you need to set `stack="diverging` and probably update your `y-scale` domain logic. Your sandbox didn't have any negative values, but here's one that does (note the y-scale domain logic isn't technically correct/could use improvements)

https://codesandbox.io/s/naughty-aryabhata-y7s051?file=/Example.tsx

<img width="677" alt="image" src="https://user-images.githubusercontent.com/4496521/211077675-211051e7-6e92-4aba-b27a-620560e41528.png">
